qtawesome/iconic_font.py

```diff
@@ -234,11 +233,11 @@ def hook(obj):
             result = {}
             for key in obj:
                 try:
-                    result[key] = unichr(int(obj[key], 16))
+                    result[key] = chr(int(obj[key], 16))
                 except ValueError:
                     if int(obj[key], 16) > 0xffff:
```

Comment:
I think I can also remove this, right @ccordoba12 ?

Reply:
I'm not sure, sorry.